Sources/Extensions/SwiftStdlib/BoolExtensions.swift

```diff
@@ -35,5 +35,14 @@ public extension Bool {
 	public static var random: Bool {
 		return arc4random_uniform(2) == 1
 	}
+    
+    /// SwifterSwift: Returns a toogled boolean value.
+    ///
+    ///      var on = true
+    ///      on = on.toogle() -> false
+    ///
+    public mutating func toggle() {
```

Comment:
I think that `toggle` is not necessary. It is native in Swift 4.2 [SE-199](https://github.com/apple/swift-evolution/blob/master/proposals/0199-bool-toggle.md)